docs/content/preview/manage/ysql-major-upgrade-yugabyted.md

```diff
@@ -0,0 +1,221 @@
+```
+
+```sh
+./path_to_new_version/bin/yugabyted start \
+    --base_dir=~/yugabyte-data/node1 \
+    --tserver_flags="ysql_yb_major_version_upgrade_compatibility=11" \
+    --master_flags="ysql_yb_major_version_upgrade_compatibility=11"
+```
+
+{{< warning title="Enable Mixed Mode" >}}
+Restart nodes with the flag `ysql_yb_major_version_upgrade_compatibility=11` so that the universe can function in mixed mode, with both PostgreSQL 15 and (yet to be upgraded) PostgreSQL 11 nodes.
+{{< /warning >}}
+
+```output
+Starting yugabyted...
+✅ Upgrade status successfully verified   
+✅ YugabyteDB Started                  
+✅ Node joined a running cluster with UUID 5dd3bda3-43a9-48fd-9c16-8399378fed12
+✅ UI ready         
+✅ Data placement constraint successfully verified                 
+
++---------------------------------------------------------------------------------------------------+
+|                                               yugabyted                                           |
++---------------------------------------------------------------------------------------------------+
+| Status              : Running.                                                                    |
+| YSQL Status         : Ready                                                                       |
+| Replication Factor  : 3                                                                           |
+| YugabyteDB UI       : http://127.0.0.1:15433                                                      |
+| JDBC                : jdbc:postgresql://127.0.0.1:5433/yugabyte?user=yugabyte&password=yugabyte   |
+| YSQL                : bin/ysqlsh -h 127.0.0.1  -U yugabyte -d yugabyte                            |
+| YCQL                : bin/ycqlsh 127.0.0.1 9042 -u cassandra                                      |
+| Data Dir            : /net/dev-server-hsunder/share/yugabyte-data/node1/data                      |
+| Log Dir             : /net/dev-server-hsunder/share/yugabyte-data/node1/logs                      |
+| Universe UUID       : 5dd3bda3-43a9-48fd-9c16-8399378fed12                                        |
++---------------------------------------------------------------------------------------------------+
+```
+
+## Monitor phase
+
+After all the YB-Master and YB-TServer processes are upgraded, monitor the cluster to ensure it is healthy. Make sure workloads are running as expected and there are no errors in the logs.
+
+You can remain in this phase for as long as you need, but you should [finalize the upgrade](#finalize-phase) sooner rather than later to avoid operator errors that can arise from having to maintain two versions.
+
+DDLs are not allowed even in this phase. New features that require format changes will not be available until the upgrade is finalized. Also, you cannot perform another upgrade until you have completed the current one.
+
+### Disable mixed mode
```

Comment:
This should be the last step of the Upgrade Phase. It should happen before Monitor Phase